src/main/java/frc/robot/util/ProntoSwerveModule.java

```diff
@@ -113,6 +120,10 @@ public SwerveModulePosition getPosition() {
         );
     }
 
+    /**
+     * 
+     * @return motor speed & current motor rotation
```

Comment:
a SwerveModuleState contains that velocity of the wheel (not the motor) in m/s and the direction of the wheel as a Rotation2d